cxx-qt-gen/src/writer/rust/mod.rs

```diff
@@ -6,6 +6,54 @@
 use crate::generator::rust::GeneratedRustBlocks;
 use proc_macro2::TokenStream;
 use quote::{format_ident, quote, ToTokens};
+use syn::{Ident, Item};
+
+/// Add common methods into the CXX bridge which the C++ writer adds as well
+fn write_cxx_common_methods(
+    content: &mut Vec<Item>,
+    cpp_struct_ident: &Ident,
+    rust_struct_ident: &Ident,
+    namespace_internals: &String,
+) {
+    content.push(
+        syn::parse2(quote! {
+            unsafe extern "C++" {
```

Comment:
Should this also contain the convert include?
``` rust
include!("cxx-qt-lib/include/convert.h");
```

Reply:
Done